lib/defaults.go

```diff
@@ -27,8 +23,11 @@ func GetDefaultBin() string {
 const (
 	DefaultMirror             = "https://releases.hashicorp.com/terraform"
 	DefaultLatest             = ""
+	distTerraform             = "terraform"
+	distTofu                  = "tofu"
```

Comment:
distribution, I'm assuming? Though, I wonder if people would get annoyed if we called Terraform/OpenTofu distributions, since it's sort of implies two different variants/releases from a common source (at least, my interpretation) :D

Reply:
yes distribution it is. just needed a variable name. if you have something better i am all ears 😁

Reply:
Maybe when @MatthewJohn gets this PR incorporated into #435 we can use https://github.com/warrensbox/terraform-switcher/blob/315-opentofu-support/lib/products.go for this too? 🤔 
Otherwise I'd think of something like `binTerraform` and `binOpenTofu` (not ideal but at least eliminates `dist` confusion for me 🤷🏻)

Reply:
> (not ideal but at least eliminates `dist` confusion for me 🤷🏻)

I'm assuming myself as one of the ordinary users that sometimes look into code 😺

Reply:
> Maybe when @MatthewJohn gets this PR incorporated into https://github.com/warrensbox/terraform-switcher/pull/435 we can use https://github.com/warrensbox/terraform-switcher/blob/315-opentofu-support/lib/products.go for this too? 🤔

Yep :)